client/verta/verta/runtime.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
fyi if you weren't intending to already, we ought to add a module docstring ([example](https://github.com/VertaAI/modeldb/blob/7d7d578a5f09d40f90b374b369969ecb5610c018/client/verta/verta/credentials.py#L2-L6)) and add `runtime` to [python.rst](https://github.com/VertaAI/modeldb/blob/main/client/verta/docs/python.rst).

Reply:
Absolutely.  Will do

Reply:
Done. Can you sanity check for me that I have added the `runtime` module to `python.rst` correctly?